File: Java/topflytechCodec/src/com/topflytech/codec/entities/NetworkInfoMessage.java
package com.topflytech.codec.entities;

import java.util.Date;

/**
 * Created by admin on 2018/9/21.
 */
public class NetworkInfoMessage extends Message {
    public String getNetworkOperator() {
        return networkOperator;
    }

    public void setNetworkOperator(String networkOperator) {
        this.networkOperator = networkOperator;
    }

    public String getAccessTechnology() {
        return accessTechnology;
    }

    public void setAccessTechnology(String accessTechnology) {
        this.accessTechnology = accessTechnology;
    }

    public String getBand() {
        return band;
    }

    public void setBand(String band) {
        this.band = band;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public String getImsi() {
        return imsi;
    }

    public void setImsi(String imsi) {
        this.imsi = imsi;
    }

    public String getIccid() {
        return iccid;
    }

    public void setIccid(String iccid) {
        this.iccid = iccid;
    }

    public String getWifiMac() {
        return wifiMac;
    }

    public void setWifiMac(String wifiMac) {
        this.wifiMac = wifiMac;
    }

    public String getWifiSsid() {
        return wifiSsid;
    }

    public void setWifiSsid(String wifiSsid) {
        this.wifiSsid = wifiSsid;
    }

    private String networkOperator;
    private String accessTechnology;
    private String band;
    private String imsi;
    private Date date;
    private String iccid;
    private String wifiSsid;
    private String wifiMac;

}

File: C#/TopflytechCodec/TopflytechCodec/Entities/NetworkInfoMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    public class NetworkInfoMessage : Message
    {
        private String networkOperator;

        public String NetworkOperator
        {
            get { return networkOperator; }
            set { networkOperator = value; }
        }
        private String accessTechnology;

        public String AccessTechnology
        {
            get { return accessTechnology; }
            set { accessTechnology = value; }
        }
        private String band;

        public String Band
        {
            get { return band; }
            set { band = value; }
        }
        private String imsi;

        public String Imsi
        {
            get { return imsi; }
            set { imsi = value; }
        }
        private DateTime date;

        public DateTime Date
        {
            get { return date; }
            set { date = value; }
        }
        private String iccid;

        public String Iccid
        {
            get { return iccid; }
            set { iccid = value; }
        }

        private String wifiSsid;
        public String WifiSsid
        {
            get { return wifiSsid; }
            set { wifiSsid = value; }
        }
        private String wifiMac;
        public String WifiMac
        {
            get { return wifiMac; }
            set { wifiMac = value; }
        }
    }
}
